docs/modules/ROOT/pages/push.adoc

```diff
@@ -0,0 +1,94 @@
+== AeroGear JavaScript Unifiedpush SDK
+
+An SDK that allows you to receive push notifications and integrate with the Aerogear Unifiedpush Server. In contrast to the other SDKs, `@aerogear/push` itself does not implement the handling of push notifications. It helps you to integrate a third party push library with our backend, the Aerogear Unifiedpush Server. While you can use any push plugin you like that supports Firebase Cloud Messaging and Apple Push Notification Services we suggest link:https://github.com/phonegap/phonegap-plugin-push[Phonegap Plugin Push] for Cordova and Ionic Apps.
```

Comment:
One thing that really sticks out like a sore thumb is the sentence `It helps you to integrate a third party push library with our backend, the Aerogear Unifiedpush Server.` The use of the word `backend` here is especially misleading.

To be honest I find this entire summary to be quite confusing. I would be more straight to the point and I would try to break it up. It's a pretty large paragraph.

Something like this:


== AeroGear JavaScript Unifiedpush SDK

`@aerogear/push` makes it simple to receive push notifications in Cordova and Ionic Apps. It is a thin wrapper that extends the popular link:https://github.com/phonegap/phonegap-plugin-push[Phonegap Plugin Push] library to also work with Aerogear Unifiedpush Server.

It can be integrated with Aerogear Unifiedpush Server or it can be used directly with Firebase Cloud Messaging and Apple Push Notification Service.

=== Supported Platforms

* Android
* iOS

Reply:
Yep, thats better. Going to use this.